File: web/src/model/victron_controller/dashboard/from_0_2_0_soc-history-sample.ts
// @ts-nocheck
import {SocHistorySample as v0_2_0_SocHistorySample} from './v0_2_0/SocHistorySample'
import {SocHistorySample as dashboard_SocHistorySample} from './SocHistorySample'

export function convert__soc_history_sample__from__0_2_0(from: v0_2_0_SocHistorySample): dashboard_SocHistorySample {
    return new dashboard_SocHistorySample (
        from.epoch_ms,
        from.soc_pct
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_soc_history_sample.rs


pub fn convert__soc_history_sample__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::soc_history_sample::SocHistorySample) -> crate::victron_controller::dashboard::soc_history_sample::SocHistorySample {
    crate::victron_controller::dashboard::soc_history_sample::SocHistorySample {
        epoch_ms: from.epoch_ms.clone(),
        soc_pct: from.soc_pct.clone(),
    }
}